plugins/pip/venvShellHook.sh

```diff
@@ -0,0 +1,9 @@
+SOURCE_DATE_EPOCH=$(date +%s)
```

Comment:
what's this?

Reply:
Based this on the Nix manual, but looking deeper i think this is only needed for python2. I'll remove it